app/controllers/forms_controller.rb

```diff
@@ -5,6 +5,7 @@ class FormsController < ApplicationController
 
   def show
     @form = Form.find(params[:id])
+    @pages = @form.pages
```

Comment:
why do we need a new instance var? when we can use `@forms.pages` in the view itself?